docs/rules/fenced-code-meta.md

```diff
@@ -0,0 +1,76 @@
+# fenced-code-meta
+
+Require or disallow metadata for fenced code blocks.
+
+## Background
+
+Fenced code blocks can include an info string after the opening fence. The first word typically specifies the language (e.g., `js`). Many tools also support additional metadata after the language (separated by whitespace), such as titles or line highlighting parameters. This rule enforces a consistent policy for including such metadata.
```

Comment:
```suggestion
Fenced code blocks can include an information string after the opening fence. The first word typically specifies the language (e.g., `js`). Many tools also support additional metadata after the language (separated by whitespace), such as titles or line highlighting parameters. This rule enforces a consistent policy for including such metadata.
```

Reply:
We should stick with "info string":
https://spec.commonmark.org/0.31.2/#info-string

@TKDev7 when referencing the spec, go ahead and include a link to the section you're referring to. That helps move the conversation forward.